src/seg_utils.py

```diff
@@ -344,6 +344,7 @@ def segment(net, device, inference_loader, qlty_object, tiled_client):
             batch = batch.to(device)
             # Input passed through networks here
             patch_output = net(batch)
+            patch_output = patch_output.cpu()
             stitched_result, weights = qlty_object.stitch(patch_output)
             # Individual output passed through argmax to get predictions
             seg_batch = torch.argmax(stitched_result.cpu(), dim=1).numpy().astype(np.int8)
```

Comment:
I think you can remove `.cpu()` from here

Reply:
But right now the `patch_output` is still in cuda, won't this running into issue between devices?

Reply:
Addressed based on discussion.